src/coreclr/pal/src/thread/context.cpp

```diff
@@ -623,10 +652,17 @@ void CONTEXTToNativeContext(CONST CONTEXT *lpContext, native_context_t *native)
         static_assert_no_msg(sizeof(fp->fprs) == sizeof(lpContext->Fpr));
         memcpy(fp->fprs, lpContext->Fpr, sizeof(lpContext->Fpr));
 #elif defined(HOST_LOONGARCH64)
+        native->uc_mcontext.__fcsr = lpContext->Fcsr;
```

Comment:
@shushanhf, this was the missing CSR case I was referring to. Added it here while I was touching this file for riscv64 case below.

Reply:
Ok, thanks very much.